salt/cni/kube-flannel.yaml.jinja

```diff
@@ -46,6 +46,9 @@ data:
       {%- endif %}
       "Backend":
       {
+        {%- if salt.caasp_pillar.get('flannel:backend') in "vxlan", "udp" %}
```

Comment:
Is this valid syntax? don't we need `if salt.caasp_pillar.get('flannel:backend') in ("vxlan", "udp")` ?